src/utils/methods.js

```diff
@@ -0,0 +1,25 @@
+export function validateEmail(value) {
+  if (!value) {
+    return 'Email is required';
+  }
+
+  const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailPattern.test(value)) {
+    return 'Email is not valid';
+  } else {
+    return 'email is valid';
```

Comment:
The message 'email is valid' is inconsistent with the other validation messages which start with a capital letter. Consider changing it to 'Email is valid' for consistency.